### ISSUE TITLE
Upload does not work for pg_back_2.0.1_linux_amd64.deb

### ISSUE THREAD

**Issue by vm-mishchenko** [orgrim/pg_back#69]
Hi,
I downloaded the `pg_back_2.0.1_linux_amd64.deb` file from the release page and install it on Ubuntu 21.10 using:
```
sudo dpkg -i ./pg_back_2.0.1_linux_amd64.deb
```

Looks like `pg_back` does not recognize that `upload` option is set and as result does not upload backups to cloud (`gcs` in my case). There is no errors about misconfiguration.

But it does work if I compile the binary using:
```
docker run --rm -v "$PWD":/go/bin golang:1.16 go get github.com/orgrim/pg_back
```
In this case everything is working with the exact same config file I used previously.

**Comment by orgrim** [comment]
Hello,

This feature will be available in version 2.1.0, and it is not released yet. When you use go get, the development version is built, not v2.0.1, this why it works.

Concerning the fact that there is no information that the parameters you added in the configuration file are recognized, issue #65 reports the problem and PR #68 fixes it.

Regards

**Comment by vm-mishchenko** [comment]
ou my bad, thanks for explanation. It totally makes sense.
